2018 Robot/src/org/usfirst/frc/team4999/robot/RobotMap.java

```diff
@@ -40,10 +40,13 @@
 	public static VictorSP leftBackMotor = new VictorSP(3);
 	public static VictorSP rightFrontMotor = new VictorSP(0);
 	public static VictorSP rightBackMotor = new VictorSP(1);
-	
+	public static double auto_speed = 0.25;
```

Comment:
This should either be in the robot preferences or in a static field on the smartdashboard